### PR TITLE
PMM-5081: prometheus metrics for inventory data

### PR DESCRIPTION
**Pull request by ritbl**
[PMM-5081](https://jira.percona.com/browse/PMM-5081)

same as https://github.com/percona/pmm/pull/1854 but from this repo

FB: https://github.com/Percona-Lab/pmm-submodules/pull/3292

### CODE REVIEW

File: api/inventorypb/agent_status.proto [hunk 1]
```diff
@@ -7,16 +7,16 @@ option go_package = "api/inventorypb;inventorypb";
...
+  WAITING = -2;
   // Agent is stopping.
-  STOPPING = 4;
+  STOPPING = -3;
   // Agent finished.
-  DONE = 5;
+  DONE = -4;
   // Agent is not connected, we don't know anything about it's state.
-  UNKNOWN = 6;
+  UNKNOWN = -5;
```
Comment (BupycHuk) [inline api/inventorypb/agent_status.proto:21]: it's a breaking change. please revert it.

**Review by BupycHuk — CHANGES_REQUESTED** [review]
most of changes in `api` directory in this PR is unnecessary, please revert them. Also please revert `descriptor.bin`. it's made to not break backward compatibility. 
And in https://github.com/percona/pmm/pull/1854#pullrequestreview-1449372898 there were a really good review from @artemgavrilov, so please address them as well.

File: api/inventorypb/agent_status.proto (hunk 1 above)
Comment (ritbl) [reply api/inventorypb/agent_status.proto]: done

**Comment by ritbl** [comment]
@BupycHuk done, please check

**Comment by ritbl** [comment]
@BupycHuk update without addressing https://github.com/percona/pmm/pull/1854#discussion_r1209257656

**Review by idoqo — APPROVED** [review]

**Pull request merged**